libqtile/confreader.py

```diff
@@ -99,6 +99,25 @@ def update(self, *, fake_screens=None, **settings):
                 value = getattr(self, key, default[key])
             setattr(self, key, value)
 
+    def reload_config_submodules(self, path):
+        """Reloads python files from same folder as config file."""
+        to_reload = []
+        folder = path.parent
+        for name, module in sys.modules.items():
+
+            # Skip built-ins and anything with no filepath.
+            if getattr(module, "__file__", False):
+                subpath = Path(module.__file__).parent
+
+                # Check if the module is in the config folder or subfolder
+                if folder in subpath.parents:
+
+                    # We can't reload in this loop as that will alter sys.modules
+                    to_reload.append(module)
+
```

Comment:
You could do `for name, module in sys.modules.copy().items()` so you can do it in one loop and drop `to_reload`